packages/compiler-cli/ngcc/src/main.ts

```diff
@@ -31,7 +31,7 @@ export interface NgccOptions {
   /** The absolute path to the `node_modules` folder that contains the packages to process. */
   basePath: string;
   /**
-   * The path, relative to `basePath` to the primary package to be processed.
+   * The absolute path or relative to `basePath` of the primary package to be processed.
```

Comment:
This is a bit convoluted. Can we go with something like this instead: https://github.com/angular/angular-cli/pull/14325#discussion_r280418706

```suggestion
   * The path to the primary package to be processed. If not absolute then it must be relative to `basePath`.
```